src/main/java/team/chisel/ctm/api/model/IModelCTM.java

```diff
@@ -21,6 +22,8 @@ public interface IModelCTM extends IUnbakedGeometry<IModelCTM> {
 
     boolean canRenderInLayer(BlockState state, RenderType layer);
 
+    Set<RenderType> getExtraLayers(BlockState state);
```

Comment:
Should `canRenderInLayer` be deprecated?

Reply:
I believe so, in fact I think it should be deprecated as for removal, but I held off on it as I figured it could be addressed when the question I asked in the PR description about defaulting the new `getExtraLayers` method and also how you want to handle breaking changes. There is also the fact that it returns a faulty value for purposes of things like sugar cane in your test pack as `canRenderInLayer` returns true for solid there.